package/com.unity.formats.usd/Documentation~/USD Export.md

```diff
@@ -0,0 +1,77 @@
+# Exporting USD files
+
+USD files can be created by exporting GameObjects selected in the Hierarchy using the USD context menu options, or using the Recorder window. Supported USD file formats are:
+
+* USDC (.usd or .usdc)
+* USDA (.usda)
+* USDZ (.usdz)
+
+The default file format when exporting is .usd with USDC encoding, but changing the file ending in the save menu to .usda will export as USDA.
+
+## Export Selected with Children
+
+On Mac you will see multiple file format options- USDC and USDA.
+
+On Windows the default is .usd with a USDC format, but you can change the file ending in the Save window to specify a different file ending and encoding.
+
+
+## Export Transform Overrides
+
+Modifications to transforms in USD files can be exported as overrides to the original files. 
+
+Overrides can be exported from the USD menu with the GameObject containing the [UsdAsset](/USD-in-the-Editor#USD-Asset) component selected, which will export transform overrides for the entire hierarchy. Alternatively, you can also export just the overrides when exporting from the Recorder window by changing the 'Override Setting' to 'Export Transform Overrides Only'.
```

Comment:
Dead-link: `/USD-in-the-Editor#USD-Asset` is does not work from Github (https://github.com/Unity-Technologies/usd-unity-sdk/blob/documentation/rewrite/package/com.unity.formats.usd/Documentation~/USD%20Export.md)

I think removing the leading '/' (root folder) shall resolve this as the file is in the same folder.

Reply:
Thanks!